Adds note about setting transition-days flag to 0 (#1420)
minio/docs: source/reference/minio-mc/mc-ilm-rule-add.rst

   The number of calendar days from object creation after which MinIO marks an object as eligible for transition. 
   MinIO transitions the object to the configured remote tier specified to the :mc-cmd:`~mc ilm rule add --transition-tier`. 
   Specify the number of days as an integer, e.g. ``30`` for 30 days.

>    If the remote tier is another MinIO deployment, transition days can be set to ``0`` to mark new objects for the quickest move to the remote tier.

Reviewer: wording

```suggestion
   If the remote tier is another MinIO deployment, you can set the value to to ``0`` to mark new objects as immediately eligible for transition to the remote tier.
```

Removes passive, and emphasizes eligibility
